src/daml/Daml/Trading/Cash/CreditCashAccount.daml

```diff
@@ -35,3 +35,45 @@ template Request
     choice Withdraw : ()
       controller owner
       do pure ()
+
+
+template UserRequest
+  with
+    owner: Party 
+    custodian: Party 
+    instrument : Instrument.K
+    amount : Decimal
+    holdingCid : ContractId Holding.I
```

Comment:
move this to Credit choice's param. User cannot know the holdingCid of admin when creating request